app/assets/javascripts/initializers/initializeSheCodedArticles.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// This file is specifically used for SheCoded,
+// We can try to make it more general for campaigns in the future once we know more (classnames, params).
+
+function renderArticles(articles, articleElement) {
+  let articleDiv = articleElement;
+  const articleHTML = articles.map((article) => {
+    return `<a class="she-coded-page__some-story-container-link" href="${article.url}"><div class="she-coded-page__some-story-container">
+      <div class="she-coded-page__some-story-tag"><span class="she-coded-page__some-story-hashtag">#</span>shecoded</div>
+      <div class="she-coded-page__some-story-header">${article.title}</div>
+      <div class="she-coded-page__some-story-info">
+        <div class="she-coded-page__some-story-info-author-details">
+          <div class="she-coded-page__some-story-info-pic">
+            <img src="${article.user.profile_image}" alt="${article.user.name}'s profile picture"/>
+          </div>
+          <div class="she-coded-page__some-story-info-name">${article.user.name}</div>
+        </div>
+        <div class="she-coded-page__some-story-info-date">&nbsp; &#183; &nbsp; ${article.readable_publish_date}</div>
+      </div>
+    </div></a>`
+  });
+
+  articleDiv.innerHTML = articleHTML.join(" ");
```

Comment:
I'm pretty sure react expects `dangerouslySetInnerHTML` here. Some relevant [docs](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) here.

Reply:
this is not a component though, it's "free form", old school javascript :)

Reply:
@vaidehijoshi we're not using preact here, its just a plain js initializer

Reply:
Ahhhh, whoops! You're right. Please ignore this, entirely.